OSDOCS CQA WINC-3: Windows Containers Release Notes and Support (#113586)
openshift/openshift-docs: windows_containers/wmco_rn/windows-containers-release-notes-limitations.adoc

= Windows Machine Config Operator known limitations
include::_attributes/common-attributes.adoc[]

[role="_abstract"]

> You should familiarize yourself with the set of known limitations when working with Windows nodes managed by the WMCO (Windows nodes).

Reviewer: ```suggestion
Familiarize yourself with the set of known limitations when working with Windows nodes managed by the WMCO (Windows Machine Config Operator).
```
working with Windows nodes that are managed by Windows nodes? should this be the Operator?

> Note the following limitations when working with Windows nodes:

Reviewer: tbh i think you could remove this sentence and unbullet the top-level ones

Author: I tried this, but it seems easier to parse as a list. Maybe just me; but with each limitation as a paragraph, not a bullet, the page seemed denser.
